Source/ALSV4_CPP/Private/Character/ALSBaseCharacter.cpp

```diff
@@ -96,6 +97,7 @@ void AALSBaseCharacter::BeginPlay()
 
 	// Update states to use the initial desired values.
 	SetGait(DesiredGait);
+	SetDesiredStance(DesiredStance);
```

Comment:
Indentation problem here

Reply:
I don't see the issue here, all the lines from line 71 to 106 use a single Tab, and I see no reason it should be any different.

I don't understand, am I overlooking something?

Reply:
It's probably a problem with Github Android app then, it shows that line with a tab on the beginning :)